cmd/symbols/gitserver/client.go

```diff
@@ -101,8 +101,30 @@ func (c *gitserverClient) LogReverseEach(ctx context.Context, repo string, commi
 	return c.innerClient.LogReverseEach(ctx, repo, commit, n, onLogEntry)
 }
 
+const revListPageSize = 100
+
 func (c *gitserverClient) RevList(ctx context.Context, repo string, commit string, onCommit func(commit string) (shouldContinue bool, err error)) error {
-	return c.innerClient.RevList(ctx, repo, commit, onCommit)
+	nextCursor := commit
+	for {
+		var commits []api.CommitID
+		var err error
+		commits, nextCursor, err = c.innerClient.RevList(ctx, api.RepoName(repo), nextCursor, revListPageSize)
```

Comment:
Can this be an in-line assignment instead of a declaration and then assignment? 


```suggestion
commits, nextCursor, err := c.innerClient.RevList(ctx, api.RepoName(repo), nextCursor, revListPageSize)
```

Reply:
This would create a block-scoped `nextCursor` so it wouldn't update that one - I could do a `nc` and then a second line with `nextCursor = nc` if that's prettier?

Reply:
Ah, I see. Don't worry about it, I think this looks fine